msal/src/Microsoft.Identity.Client/UIParent.cs

```diff
@@ -112,24 +114,12 @@ public UIParent(Activity activity, bool useEmbeddedWebview) : this(activity)
         /// </example>
         public static bool IsSystemWebviewAvailable()
```

Comment:
>IsSystemWebviewAvailable [](start = 27, length = 24)

I believe your testing also found that even without custom tabs there are some browsers where getting the token still works?

Reply:
but then we are back to list. we know for sure the ones w/custom tabs works. not sure why some other browsers do work and others do not. Up for discussing this further. 

---
In reply to: [227940590](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227940590) [](ancestors = 227940590)

Reply:
we could also only allow browsers w/custom tabs support, which covers most of the popular android ones, except for a Opera and Dolphin (which doesn't work). 

---
In reply to: [227940590](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227940590) [](ancestors = 227940590)